server/routes/api/browser.py

```diff
@@ -60,3 +61,53 @@ def get_property_labels(dcid):
     """Returns all property labels given a node dcid."""
     labels = dc.get_property_labels([dcid]).get(dcid, {})
     return Response(json.dumps(labels), 200, mimetype='application/json')
+
+
+@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
+@bp.route('/observationId')
+def get_observation_id():
+    """Returns the dcid of the observation node for a combination of predicates: observedNodeLocation,
+    statisticalVariable, observationDate, measurementMethod (optional), observationPeriod (optional)"""
+    place_id = request.args.get("place")
+    if not place_id:
+        return Response(json.dumps("error: must provide a placeId field"),
```

Comment:
:s/placeId/place,  as "place" is the input argument

Reply:
done